src/Sulu/Component/Rest/ListBuilder/Metadata/Resources/schema/metadata/list-builder-general-2.0.xsd

```diff
@@ -116,5 +116,5 @@
         <xs:attribute name="key" type="xs:string"/>
     </xs:complexType>
 
-    <xs:element name="class" type="classType"/>
+    <xs:element name="datagrid" type="datagridType"/>
```

Comment:
would not use the term `datagrid` for a defining a list for datagrid

Reply:
That's what @chirimoya and I agreed one, in order to make it more consistent with the form XML. What would your suggestion be?